tests/test_api_materials.py

```
@@ -0,0 +1,662 @@
+"""
+Tests for the Course Materials API (A2 module).
+
+Covers:
+  * list_materials   – GET  /api/activities/:activity_id/materials
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)